Dockerfile

```
@@ -33,6 +33,8 @@ COPY glam /app/glam
 # FRONTEND BUILDER IMAGE
 FROM node:lts-slim AS frontend
 
+RUN apt-get update || : && apt-get install python -y
```

```suggestion
RUN apt-get update || : && apt-get install python3 -y
```

`lts-slim` seems to be based on debian buster, where the python package is just actually python 2. It's python 3 we actually want.

https://hub.docker.com/_/node
https://packages.debian.org/buster/python
https://packages.debian.org/buster/python3